sig-storage/annual-report-2024.md

```diff
@@ -12,15 +12,39 @@
    - Governance and leadership changes
 -->
 
+GA milestones hit
+
+- [3756 - Robust VolumeManager Reconstruction](https://github.com/kubernetes/enhancements/issues/3756) moved to GA in v1.30. This feature improves Kubelet handling of mounted volumes across restarts, enabling faster, more robust recovery.
+- [3141 - Prevent Unauthorized Volume Mode Conversion](https://github.com/kubernetes/enhancements/issues/3141) moved to GA in v1.30. This feature prevents unauthorized conversion of the volume mode when creating a Persistent Volume Claim from a Volume Snapshot.
+- [3762 - Persistent Volume last phase transition time](https://github.com/kubernetes/enhancements/issues/3762) moved to GA in v1.31. This feature adds a PersistentVolumeStatus field which holds a timestamp of when a PersistentVolume last transitioned to a different phase. This allows you to measure time e.g. between a PV Pending and Bound. This can be also useful for providing metrics and SLOs.
+- [1847 - Auto remove PVCs created by StatefulSet](https://github.com/kubernetes/enhancements/issues/1847) moved to GA in v1.32. This feature, co-owned with SIG-Apps, allows the PVCs created by StatefulSet to be automatically deleted when the volumes are no longer in use to ease management of StatefulSets that don't live indefinitely.
+
+Successful progress
+
+- [3751 - Kubernetes VolumeAttributesClass ModifyVolume](https://github.com/kubernetes/enhancements/issues/3751) moved to Beta in v1.31. This feature extends the Kubernetes Persistent Volume API to allow users to dynamically modify volume options (such as IOPs and throughput), after a volume is provisioned. (Note: The current status of this enhancement is marked as at risk for code freeze.)
+- [1790 - Recover from Volume Expansion Failure](https://github.com/kubernetes/enhancements/issues/1790) moved to Beta  in v1.32. This feature allows users to recover from volume expansion failure by retrying with a smaller size.
+- [3476 - Volume Group Snapshot](https://github.com/kubernetes/enhancements/issues/3476) moved to Beta in v1.32. The feature introduces a VolumeGroupSnapshot API to take a snapshot of multiple volumes together.
+
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+- Write more tests; monitor test grid health; work on test
+framework out of tree; enhance CSI release tools.
+- Doc writer to improve docs on CSI side and in general in
+Storage side.
```

Comment:
https://kubernetes-csi.github.io/docs/ the doc homepage and changelog seems to be not updated. 
- https://github.com/kubernetes/kubernetes/pull/71020 Are version matrics correct?

Reply:
Right, the changelog page has not been updated.  We need to do that.
The table on the home page shows both 0.3.0 and v1.0.0 are compatible with K8s 1.13. Is that what you are referring to?

Reply:
yeah. The PR changelog does not match that.